gradle.properties

```
@@ -1,9 +1,8 @@
 org.gradle.jvmargs=-Xmx2560M
```

Soooo, QFAPI already had it but FAPI added it recently; plus + plus != minus
```suggestion
org.gradle.jvmargs=-Xmx2560M
org.gradle.parallel=true
```